src/shared/user-agent.js

```diff
@@ -10,3 +10,28 @@
 export const isMacOS = (_userAgent = window.navigator.userAgent) => {
   return _userAgent.indexOf('Mac OS') >= 0;
 };
+
+/**
+ * Returns true when device is iOS.
+ * https://stackoverflow.com/a/9039885/14463679
+ *
+ * @param _navigator {{platform: string, userAgent: string}}
+ * @param _ontouchend {boolean}
+ */
+export const isIOS = (
+  _navigator = window.navigator,
+  _ontouchend = 'ontouchend' in document
+) => {
```

Comment:
You can use ` /* istanbul ignore next */` to suppress the code coverage check for the default parameter initialization here. See https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md. Alternatively, add a test which just executes `isIOS` without passing test seams.